### PR TITLE
Fix/mf35 sequence

### PR DESCRIPTION
**Pull request by whaeck**
Properly deallocating sammy arrays solves the issue when running multiple ERRORR isntances in the same input.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Given here are some release notes for NJOY2016. Each release is made through a f
 ## [NJOY2016.70](https://github.com/njoy/NJOY2016/pull/xxx)
 This update fixes a number of minor issues:
   - Fixed an issue in HEATR when reading evaluations with large multiplicity tables in MF6.
+  - Multiple ERRORR calls can now be made in the same input file without crashing. This is of interest to users that wish to process MF34 and MF35 (where ERRORR needs to be called for each sub-subsection and incident energy group).
```
Comment (nathangibson14) [inline ReleaseNotes.md:7]: You should make reference to this affecting files with LRF7 resonances without MF32 data.

**Review by nathangibson14 — DISMISSED** [review]
Just a documentation nitpick.  This works on the affected files for me, and tests seem to pass.

**Comment by whaeck** [comment]
@nathangibson14 need you to reapprove

**Comment by whaeck** [comment]
@nathangibson14 You may want to reapprove this one

**Review by nathangibson14 — APPROVED** [review]

**Pull request merged**